Description: This is a fix for the new build failure introduced by new acpica-unix,
 lets workaround it for now
 see discussion in: https://github.com/acpica/acpica/commit/f9a88a4c1cd020b6a5475d63b29626852a0b5f37#diff-95e84a7c99c268ce5433dea47cf17ec0
Author: FreeBSD team https://svnweb.freebsd.org/ports/head/emulators/virtualbox-ose/files/patch-src_VBox_Devices_Makefile.kmk?revision=469000&view=co
Bug-Debian: https://bugs.debian.org/899318
Last-Update: 2018-05-25

--- virtualbox-5.2.12-dfsg.orig/src/VBox/Devices/Makefile.kmk
+++ virtualbox-5.2.12-dfsg/src/VBox/Devices/Makefile.kmk
@@ -828,7 +828,12 @@ if !defined(VBOX_ONLY_EXTPACKS)
 
   $$(VBoxDD_0_OUTDIR)/vboxaml.hex: $(PATH_SUB_CURRENT)/PC/vbox.dsl | $$(dir $$@)
 	$(call MSG_TOOL,iasl,VBoxDD,$<,$@)
+	$(QUIET)$(RM) -f $@ $@.tmp
 	$(QUIET)$(VBOX_IASLCMD) -tc -vs -p $@ $<
+	$(QUIET)$(MV) -f $@ $@.tmp
+	$(QUIET)$(SED) -e "s/vboxaml_aml_code/AmlCode/g" \
+		--output $@ $@.tmp
+	$(QUIET)$(RM) -f $@.tmp
 
   vboxaml.hex:: $$(VBoxDD_0_OUTDIR)/vboxaml.hex
 
@@ -838,6 +843,8 @@ if !defined(VBOX_ONLY_EXTPACKS)
 	$(QUIET)$(VBOX_IASLCMD) -tc -vs -p $@ $<
 	$(QUIET)$(MV) -f $@ $@.tmp
 	$(QUIET)$(SED) -e "s/AmlCode/AmlCodeSsdtStandard/g" \
+		-e "s/__VBOXSSDT-STANDARD_HEX__/__VBOXSSDT_STANDARD_HEX__/g" \
+		-e "s/vboxssdt-standard_aml_code/AmlCodeSsdtStandard/g" \
 		--output $@ $@.tmp
 	$(QUIET)$(RM) -f $@.tmp
 
@@ -853,6 +860,8 @@ if !defined(VBOX_ONLY_EXTPACKS)
 	$(QUIET)$(VBOX_IASLCMD) -tc -vs -p $@ $@.pre1
 	$(QUIET)$(MV) -f $@ $@.tmp
 	$(QUIET)$(SED) -e "s/AmlCode/AmlCodeSsdtCpuHotPlug/g" \
+		-e "s/__VBOXSSDT-CPUHOTPLUG_HEX__/__VBOXSSDT_CPUHOTPLUG_HEX__/g" \
+		-e "s/vboxssdt-cpuhotplug_aml_code/AmlCodeSsdtCpuHotPlug/g" \
 		--output $@ $@.tmp
 	$(QUIET)$(RM) -f $@.tmp $@.pre $@.pre1
 
